pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -290,3 +302,54 @@ func (a *Account) GetCondition(conditionType AccountConditionType) *AccountCondi
 	}
 	return nil
 }
+
```

Comment:
Maybe we should consider adding an extra file for `account_conditions.go` and contain some of this logic in there?  This file is starting to get a bit long, but I'd also be open to arguments against, like keeping _all_ of the account type defs in the same file too, granted what I'm seeing below doesn't seem to be public.

Reply:
added